src/nvim/testdir/Makefile

```diff
@@ -37,7 +37,8 @@ SCRIPTS := test_autoformat_join.out                                    \
            test_qf_title.out                                           \
            test_breakindent.out                                        \
            test_insertcount.out                                        \
-           test_utf8.out
+           test_utf8.out                                               \
```

Comment:
this line can be removed, test_insertcount.out was migrated to test/functional/legacy.